source/community/release_notes/palm.rst

```diff
@@ -15,14 +15,109 @@ These are the release notes for the Palm release, the 16th community release of
 Breaking Changes
 ****************
 
+RateXBlock unsupported
+======================
+
+The RateXBlock has been deprecated and is now officially unsupported. `FeedbackXBlock 
+<https://github.com/pmitros/FeedbackXBlock>`_ is the recommended replacement.
+
+Stripe API changes
+==================
+
+Ecommerce now supports the new Stripe Payment Intents API and is no longer using the Stripe Charges API. The legacy 
+Stripe Charges ecommerce payment processor with a frontend in the ecommerce application is upgraded to a Stripe 
+Payment Intents payment processor with a frontend in frontend-app-payment. See the discussion post, `Ecommerce: 
+Changes to Stripe payment processor 
+<https://discuss.openedx.org/t/ecommerce-changes-to-stripe-payment-processor/9457>`_ for more details. 
+
+Ecommerce is still officially deprecated and we continue to work on the `Ecommerce Replacement Project 
+<https://openedx.atlassian.net/wiki/spaces/AC/pages/3617849345/Ecommerce+Deprecation+and+Replacement+Project>`_.
+
+
+Developer changes
+=================
+
+The old developer documentation site has been deprecated and removed. Please use `Open edX Developers - Latest 
+Documentation <https://docs.openedx.org/en/latest/developers/index.html>`_  instead.
+
+part of the Tutor API. This change should not affect most developers, who only use the :code:`Actions` and :code:`Filters` 
+classes (notice the plural) from :code:`tutor.hooks`. If it does affect you, there are upgrade instructions in the 
+discussion post, `Simplifying & documenting the Tutor Hooks API 
+<https://discuss.openedx.org/t/simplifying-documenting-the-tutor-hooks-api/9258>`_
+
+Most of the viewing code for PDF certificates was removed in previous releases. In the Palm release, we closed out 
+this work with an ADR (Application Decision Record) on `Leaving PDF Certificate Fields in Certificates Model 
```

Comment:
```suggestion
this work with an ADR (Architecture Decision Record) on `Leaving PDF Certificate Fields in Certificates Model 
```

Reply:
I'm not super convinced linking this ADR is that important. What information are you hoping to convey by including it?

Reply:
I think it's important to resolve the deprecation that started a couple of releases ago, and reassure folks that the legacy certificate data won't be lost, even if the functional code is removed. 

Maybe this belongs in the deprecation section, where it will be less prominent?

Reply:
I think moving to the deprecation section is a great idea.